packages/electron/src/main/modules/BacktraceMainElectronModule.ts

```diff
@@ -149,7 +149,9 @@ export class BacktraceMainElectronModule implements BacktraceModule {
     ) {
```

Comment:
there is a change from node/electron etc, is this expected?

Reply:
GitHub is drunk? The commits are on the `feature/INT-355-storage-overhaul` branch, they shouldn't be here...

Reply:
Yeah, I updated the base branch to the same branch and it's not here anymore.